client/question/components/LearnerViewDirective.js

```diff
@@ -884,10 +884,11 @@ tie.directive('learnerView', [function() {
             }
           } else {
             var feedbackParagraphs = feedback.getParagraphs();
+            var errorLine = feedback.getErrorLineNumber();
```

Comment:
For consistency call this errorLineNumber

Reply:
Done.